### PR TITLE
support build with windows

### PR DESCRIPTION
**Pull request by Chilledheart**
Under windows, official python installation path contains space such as "/c/Program Files/Python311/python".

Need escape PYTHON environment variable with quotes in bash script.

### CODE REVIEW

**Comment by klzgrad** [comment]
Simplify it, don't extend existing duct tape with more duct tapes: Use `python3` at all time.

**Comment by Chilledheart** [comment]
> Simplify it, don't extend existing duct tape with more duct tapes: Use `python3` at all time.

There is no python3 in python's official windows installer.

File: tests/basic.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 set -ex
 
 script_dir=$(dirname "$PWD/$0")
+PYTHON="$(which python3 2>/dev/null || which python 2>/dev/null)"
```
Comment (klzgrad) [inline tests/basic.sh:6]: Why repeat this

**Comment by klzgrad** [comment]
https://github.com/klzgrad/naiveproxy/commit/9ce5b852b8dd5c940c55aa574fe572d1336e6461

Always use python for windows.

**Pull request closed without merging**